tests/cases/group_test.py

```diff
@@ -33,10 +33,11 @@ class GroupTestCase(base.TestCase):
     def setUp(self):
         base.TestCase.setUp(self)
 
-        # Create a set of users so we can work with these groups
+        # Create a set of users so we can work with these groups.  User 0 is
+        # an admin
         self.users = [self.model('user').createUser(
             'usr%s' % num, 'passwd', 'tst', 'usr', 'u%s@u.com' % num)
-            for num in [0, 1, 2]]
+            for num in [0, 1, 2, 3, 4, 5]]
```

Comment:
@zachmullen, did you want me to change this, or should I merge?  (And, I agree, range is looking more appropriate.)